Source/ProgressionSystemRuntime/Public/Data/PSSaveGameData.h

```diff
@@ -54,7 +54,7 @@ class PROGRESSIONSYSTEMRUNTIME_API UPSSaveGameData : public USaveGame
 
 	/** Returns the endgame reward. */
 	UFUNCTION(BlueprintCallable, Category="C++")
-	float GetProgressionReward(EEndGameState EndGameState);
+	const float GetProgressionReward(EEndGameState EndGameState);
```

Comment:
It's gettet function, which requires:
1. Both BlueprintCallable and BlueprintPure
2. 'const' at the end, so function itself is constant

In result, in blueprints it will be displayed as green function without white execution pins

3. Finally, primitive types such as float don't need 'const' as return type or parameters, it gives absolutely nothing

Reply:
done https://github.com/moinm3uw/ProgressionSystem/pull/57/commits/8922e84569dc18493ff70dd127fe08a63ad6344a

Reply:
@moinm3uw, only 3 is resolved, problems 1 and 2 are still not addressed

Reply:
my bad: https://github.com/moinm3uw/ProgressionSystem/pull/57/commits/7f3fe20db1818af441642f48774345e5f152cd3f